src/main/java/seedu/address/logic/commands/DeleteAppointmentCommand.java

```diff
@@ -34,7 +34,7 @@ public DeleteAppointmentCommand(Index targetIndex) {
     @Override
     public CommandResult execute(Model model) throws CommandException {
         requireNonNull(model);
-        List<Appointment> lastShownList = model.getAppointmentList();
+        List<Appointment> lastShownList = model.getFilteredAppointmentList();
```

Comment:
Good usage of a similar approach to query person - makes the code easier to understand and debug.